convox.yml

```diff
@@ -3,10 +3,10 @@ services:
     build: .
     domain: ${DOMAIN}
     environment:
-      - ALGOLIA_APP
-      - ALGOLIA_INDEX
-      - ALGOLIA_KEY_ADMIN
-      - ALGOLIA_KEY_SEARCH
+      - ALGOLIA_APP=""
```

Comment:
These should just be `ALGOLIA_APP=`

Reply:
App should be modified to not attempt to upload documents to Algolia when this is not set.